styles/styles.css

```diff
@@ -888,7 +888,7 @@
   .text--body-m {
     font-size: 1rem;
     line-height: 1.5rem;
-    letter-spacing: 0.5px
+    letter-spacing: 0.005em;
   }
```

Comment:
Please copy this style over in `simplified-styles.css` as that's a smaller subset of styles that we'd eventually retain.